cmd/kpod/create_cli.go

```diff
@@ -26,3 +34,154 @@ func convertStringSliceToMap(strSlice []string, delimiter string) (map[string]st
 	}
 	return sysctl, nil
 }
+
+func verifyContainerResources(config *createConfig, update bool) ([]string, error) {
+	warnings := []string{}
+	sysInfo := sysinfo.New(true)
+
+	// memory subsystem checks and adjustments
+	if config.resources.memory != 0 && config.resources.memory < linuxMinMemory {
+		return warnings, fmt.Errorf("minimum memory limit allowed is 4MB")
+	}
+	if config.resources.memory > 0 && !sysInfo.MemoryLimit {
+		warnings = append(warnings, "Your kernel does not support memory limit capabilities or the cgroup is not mounted. Limitation discarded.")
```

Comment:
would it make sense to declare a string, append it to warnings then send it to logrus.Warn to avoid the duplications?

Reply:
Makes sense.